Bomber Guys/map.xml

```diff
@@ -153,9 +153,9 @@
   </itemremove>
   <time>5m</time>
   <broadcasts>
-    <tip after="1s" every="1m">Melon Block is breakable! Let's blow up!</tip>
+    <tip after="1s">Melon Block is breakable! Let's blow up!</tip>
     <tip after="2s">After two and a half minutes, anvils will falling down on the `8`oblack stained glass `b`ofloor! Be careful!</tip>
-    <tip after="30s" every="1m">Hey guys! Did you notice that you can reduce the explosion damage by hiding behind a pillar? Try it!</tip>
+    <tip after="30s">Hey guys! Did you notice that you can reduce the explosion damage by hiding behind a pillar? Try it!</tip>
     <alert after="140s">`6Anvils fall down after 10 seconds! Stay away from the `8black stained glass `6floor immediately!</alert>

```

Comment:
`Melon Blocks are breakable! Blow them up!`
`Did you notice that you can reduce explosion damage by hiding behind a pillar?`
``` `6Anvils fall down after 10 seconds. Stay away from the `8black stained glass `6floor immediately! ```